src/viewer.js

```diff
@@ -546,89 +538,95 @@ class VolumeImageViewer {
+            const channel = this[_channels].find(channel => {
+              return channel.blendingInformation.opticalPathIdentifier === pathIdentifier
+            })
+            if (channel) {
+              channel.addMetadata(channelMicroscopyImage.originMetadata)
             } else {
-              const opticalPathIdentifier = `${pathIdentifier}`
-              const color = [...colors[i % colors.length]]
-              const opacity = 1.0
-              const thresholdValues = [0, 255]
-              const visible = false
-              const defaultBlendingInformation = new BlendingInformation({
-                opticalPathIdentifier,
-                color,
-                opacity,
-                thresholdValues,
-                visible
-              })
-
-              const newChannel = new _Channel(defaultBlendingInformation)
-              newChannel.addMetadata(this[_options].metadata[i])
-              this[_channels].push(newChannel)
+              const blendingInformation = this[_options].blendingInformation !== undefined
+                ? this[_options].blendingInformation.find(info => {
+                    return info.opticalPathIdentifier === pathIdentifier
+                  })
+                : undefined
+              if (blendingInformation !== undefined) {
+                const newChannel = new _Channel(blendingInformation)
+                newChannel.addMetadata(channelMicroscopyImage.originMetadata)
+                this[_channels].push(newChannel)
+              } else {
+                const opticalPathIdentifier = `${pathIdentifier}`
+                const color = [...colors[i % colors.length]]
+                const opacity = 1.0
+                const thresholdValues = [0, 255]
+                const limitValues = [0, 255]
+                const visible = false
+                const defaultBlendingInformation = new BlendingInformation({
+                  opticalPathIdentifier,
+                  color,
+                  opacity,
+                  thresholdValues,
+                  limitValues,
+                  visible
+                })
+  
+                const newChannel = new _Channel(defaultBlendingInformation)
+                newChannel.addMetadata(channelMicroscopyImage.originMetadata)
+                this[_channels].push(newChannel)
+              }
             }
           }
+        } else if (channelMicroscopyImage.DimensionOrganizationType === 'TILED_SPARSE') {
+          // the spatial location of each tile is explicitly encoded using information
+          // in the Per-Frame Functional Group Sequence, and the recipient shall not
+          // make any assumption about the spatial position or optical path or order of the encoded frames.
+          throw new Error('Volume Image Viewer does hot handle TILED_SPARSE ' +
+                          'dimension organization for blending of channels yet.')
```

Comment:
I don't think it would be difficult, but I need data to look at and to use for testing. I have a dded an issue tracker in slim https://github.com/MGHComputationalPathology/slim/issues/30